public/oauth2-redirect.html

```diff
@@ -0,0 +1,89 @@
+<!--
+
+    Copyright 2018-2024 The OpenZipkin Authors
+
+    Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+    in compliance with the License. You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software distributed under the License
+    is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+    or implied. See the License for the specific language governing permissions and limitations under
+    the License.
+
+-->
+<!doctype html>
```

Comment:
was this copied from something? Also why do we need oauth of any kind as we aren't authenticating this? If we must maintain a custom build of openapi, seems we don't need to add things few will be able to maintain and also might require LICENSE notices.